src/components/Number-input/Number-input.js

```diff
@@ -95,6 +96,10 @@ NumberInput.propTypes = {
   mouseOver: PropTypes.func,
   mouseUp: PropTypes.func,
   valueChange: PropTypes.func,
+  info: PropTypes.bool,
+  clickInfo: PropTypes.func,
+  mouseDownInfo: PropTypes.func,
```

Comment:
The approach of reusing <Label /> is cool. How do we apply these interactions to the icon button element inside Label? Does setting these interactions work?

Reply:
added them separately